stock_available_to_promise_release/models/stock_move.py

```diff
@@ -499,6 +499,10 @@ def _after_release_update_chain(self):
             picking_ids.update(moves.picking_id.ids)
             moves = moves.move_orig_ids
         pickings = self.env["stock.picking"].browse(picking_ids)
+        # Don't take into account pickings that are already done or canceled
+        # This can happen if a move is a reliquat of a picking that has been
+        # already been processed.
+        # pickings = pickings.filtered(lambda p: p.state not in ("done", "cancel"))
```

Comment:
```suggestion
        pickings = pickings.filtered(lambda p: p.state not in ("done", "cancel"))
```

Reply:
Indeed commented to ensure that my test fails without this fix....